packages/components/src/Base/FC__Menu.re

```diff
@@ -16,14 +14,210 @@ external rcDividerClass: ReasonReact.reactClass = "Divider";
 
 [%bs.raw {|require("rc-menu/assets/index.css")|}];
 
+module Styles = {
+  open Css;
+  let menuBorderRadius = `px(3);
+  // Menu general, applied to menu <ul> elements
+  global(
```

Comment:
Kudos for doing this. It still seems messy (my guess is that it's a difficult area), but let's keep for now and consider improving later.

Reply:
Thanks, it's a bit messy and intrinsically messy. I've gone over it some times until being ensured everything was needed and at a somewhat right place. Still not sure of all styling needs, and was assuming some things could be switched between "global/general" styles and specific styles as style-needs become clearer.

Reply:
sounds good